core/services/vrf/delegate.go

```diff
@@ -185,18 +185,18 @@ func getStartingResponseCounts(db *gorm.DB, l logger.Logger) map[[32]byte]uint64
 	return respCounts
 }
 
-func GetStartingResponseCountsV2(db *gorm.DB, l logger.Logger) map[string]uint64 {
+func GetStartingResponseCountsV2(db *sqlx.DB, l logger.Logger) map[string]uint64 {
 	respCounts := make(map[string]uint64)
 	var counts []struct {
 		RequestID string
 		Count     int
 	}
 	// Allow any state, not just confirmed, on purpose.
 	// We assume once a ethtx is queued it will go through.
-	err := db.Raw(`SELECT meta->'RequestID' AS request_id, count(meta->'RequestID') as count
+	err := db.Select(&counts, `SELECT meta->'RequestID' AS request_id, count(meta->'RequestID') as count
```

Comment:
This query appears to be identical to the one above, can we pull it into a constant or common function?

Reply:
Done.